lib/routes/check-in/index.js

```diff
@@ -22,10 +22,17 @@ async function checkIn(server, options, done) {
       schema: schema.checkIn,
       handler: async (request, response) => {
         const { id } = request.authenticate()
-        const { ageRange, locality, ok, sex, data } = request.body
+        const {
+          ageRange,
+          locality,
+          ok,
+          sex,
+          data,
+          ...additional
```

Comment:
Should we just move all demographic data into its own db json blob?

Reply:
Could do, but would need an additional migration - I'm happy with either approach.

Reply:
Lets go with a migration, will be a cleaner and more flexible approach going forward